index.js

```diff
@@ -3,15 +3,18 @@ module.exports = leftpad;
 function leftpad (str, len, ch) {
   str = String(str);
 
-  var i = -1;
-
   if (!ch && ch !== 0) ch = ' ';
 
   len = len - str.length;
+  if (len <=0) return str;
 
-  while (++i < len) {
-    str = ch + str;
+  ch = ch + '';
+  pad_str = '';
```

Comment:
And it's better to use camel-style, let's say `var padStr`. Maybe *pad* is another choice.